styleguide/git.md

```diff
@@ -11,10 +11,9 @@ Joinbox uses a more specific version of the git-flow branching model
 - feature branches may only merged into the develop branch if they are tested and work correctly
 - for each iteration (release cycle, milestone) a new release branch is created from develop as soon everything is ready.
 - the release branch is published to a testing server for acceptance tests by the client. the client should receive the change log of the release branch
-- fixes for the release branch are merged into the release branch
-- the release branch may be merged into develop or another release branch if it is required
-- hot fixes are only created if they must be applied to the master branch for immediate release. they are based on the master branch
-- after a hot fix was released the master branch is merged into the develop branch
+- fixes for the release branch are merged into the release branch. Branches are bases on the HEAD of the release branch
+- the release branch may be merged , after the live release, into develop- and master-branch. If needed also to another release branch.
+- hot fixes are only created if they must be merge into to the master- and develop-branch for immediate release. they are based on the master branch
```

Comment:
in my vocabulary a hotfix can only exist for stuff that is released already, thus exclusively the master branch.

Reply:
how would the fix be available in the develop then, if we don't merge it into master and develop?